plugin/mutate/test/test_admin.d

```diff
@@ -0,0 +1,83 @@
+/**
+Copyright: Copyright (c) 2019, Niklas Pettersson. All rights reserved.
+License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+Author: Niklas Pettersson (nikpe353@student.liu.se)
+*/
+module dextool_test.test_admin;
+
+import std.path : relativePath;
+
+import dextool.plugin.mutate.backend.database.standalone;
+import dextool.plugin.mutate.backend.database.type;
+import dextool.plugin.mutate.backend.type;
+static import dextool.type;
+
+import dextool_test.utility;
+
+string[] errorOrFailure = ["error", "Failure"];
+
+// dfmt off
+@(testId ~ "shall mark a mutant without failing")
+unittest {
+    // arrange
+    mixin(EnvSetup(globalTestdir));
+    makeDextoolAnalyze(testEnv).addInputArg(testData ~ "fibonacci.cpp").run;
+
+    // act
+    auto r = makeDextoolAdmin(testEnv)
+        .addArg(["--operation", "markMutant"])
+        .addArg(["--id", "12"])
+        .addArg(["--to-status", "killed"])
+        .addArg(["--rationale", `"A good rationale"`])
+        .run;
+
+    // assert
+    testAnyOrder!SubStr(["Failure when marking mutant: 12"]).shouldNotBeIn(r.stderr);
```

Comment:
this is very "problematic" tests. there should be at least one that is a positiv test, in this case "shouldBeIn".
By experience a test that only check for the negative case is easy to get "out of sync" with the implementation in some way or another because as long as something do "not" happen it passes.